java/src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -71,6 +72,7 @@
     private static Model shaclModel;
     private static Model nistModel;
     private static Model nistHypoModel;
```

Comment:
Is this hypothesis mode still being used?

Reply:
removed